src/main/xar-resources/data/kwic/kwic.xml

```diff
@@ -165,5 +165,8 @@
         <para> <literal>util:expand</literal> returns a copy of the XML fragment it received in its
             first parameter, which, unless configured otherwise, has all matches wrapped into
             <tag>exist:match</tag> tags.</para>
+
+        <para>Please note that <literal>util:expand</literal> will not expand matches in Lucene fields. 
+        Use <literal>ft:highlight-field-matches</literal> instead.</para>    
```

Comment:
One additional sentence pointing to the great description of the differences between the two functions would be helpful to add here: 

> For more information, see `lucene.xml#expand-fields`

Reply:
Thanks, added now.